metricbeat/module/vsphere/datastore_usage/datastore_usage_test.go

```diff
@@ -0,0 +1,10 @@
+package datastore_usage
+
+import (
+	"github.com/stretchr/testify/assert"
+	"testing"
+)
+
+func TestFetchEventContents(t *testing.T) {
```

Comment:
Left over / missing? Any idea how we could add tests for vsphere here? What is the format of the output provided by vsphere?